docs/contributor-guide/developer-guide/dev-azure.md

```diff
@@ -1,67 +1,104 @@
-Please remember to tear down your dev instance as it does cost money.
+
+The main difference between the dev azure deploy is that it doesn't use a shared
+state (since we don't want to share state among devs). Please remember
+to [tear down](#tearing-down) your dev instance as it does cost money.
+
 # Running the dev deploy
-## Go to the civiform dev directory 
-All of these are run within the main repo and the files can be found
-in cloud/deploys/dev_azure.
+
+## Go to the civiform dev directory
+
+All of these are run within the main repo and the files can be found in
+cloud/deploys/dev_azure.
 
 ## Copy the civiform_config.example.sh
-Copy the civiform_config.example.sh into civiform_config.sh and
- change the required variables.
+
+Copy the civiform_config.example.sh into civiform_config.sh and change the
+required variables.
 
 ## Get access to correct technologies
+
 You will need to reach out to team members to get accounts for the following:
-- Azure
+
+- Azure (must be added as a contributor AND owner)
 - AWS
 - Login Radius Civiform-Staging
 
 ## Setup Local Machine
-Run through the doctor script to make sure you have the right things on
-your machine:
+
+Run through the doctor script to make sure you have the right things on your
+machine:
 
 ```
 cloud/shared/bin/doctor
 ```
 
 ## Setup Login Radius For Local Development
-Go to [Login Radius Dashboard](https://dashboard.loginradius.com/) and click
-configure a civiform integration. Choose the outbound SSO Saml.
 
-From there add an app with Sp initiated login and pick a name (this gets put
+Go to [Login Radius Dashboard](https://dashboard.loginradius.com/) and click the
+existing civiform-staging app. Click into the Integration tab, and add a new
+civiform integration. Choose the outbound SSO Saml.
+
+From there add an app with "Sp initiated login" and pick a name (this gets put
 into the config as `LOGIN_RADIUS_SAML_APP_NAME`).
 
-To generate the private key for the form run, cat the file and put into dashboard.
+To generate the private key for the form, run the following commands. Paste the
+entire output into the dashboard under "Id Provider Certificate Key". This
+creates a file in your current directory with your own private key, so make sure
+you are not accidentally adding and pushing this to the GitHub repo.
+
 ```
 openssl genrsa -out private.key 2048
+cat private.key
 ```
 
-For generating the cert run, cat the file and put into dashboard.
+For generating the cert, run the following commands. Paste the output into the
+dashboard "Id Provider Certificate".
+
 ```
 openssl req -new -x509 -key private.key -out certificate.cert -days 365 -subj /CN=civiform-staging.hub.loginradius.com
+cat certificate.cert
 ```
 
-We need to copy the details from a previous working setup in login radius
-once we set up the certs so look back at the staging one to fill out.
+"Service Provider Certificate" will be generated later while running setup.
```

Comment:
Thanks for updating these instructions! I was following them while setting up LoginRadius for myself.

This is a required field when generating the initial provider. Should the instructions be amended to include using an existing value for this until a new one is generated while running setup later?

Reply:
Ah good point, thank you! When I set up, I had originally copied the staging value over, since I didn't realize it was coming from the script.

Updated the text here.

Reply:
Great! Also, I hadn't seen this yet and was able to get it to pass validation with just having the following:

-----BEGIN CERTIFICATE-----
-----END CERTIFICATE-----

Reply:
Oh interesting, I didn't realize there was validation. Thank you! Updated the instructions.